src/replication/state.cpp

```diff
@@ -62,8 +62,10 @@ ReplicationState::ReplicationState(std::optional<std::filesystem::path> durabili
   }
 #endif
   if (std::holds_alternative<RoleReplicaData>(replication_data)) {
-    spdlog::trace("Recovered main's uuid for replica {}",
-                  std::string(std::get<RoleReplicaData>(replication_data).uuid_.value()));
+    std::string uuid = std::get<RoleReplicaData>(replication_data).uuid_.has_value()
```

Comment:
fetch just once from variant and reuse

Reply:
Where do  I fetch the second time? On line 70 it is a different variant

Reply:
Aha, Okay I got it